sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/EventHubProducerClient.cs

```diff
@@ -370,53 +370,64 @@ public virtual async Task<string[]> GetPartitionIdsAsync(CancellationToken cance
         }
 
         /// <summary>
-        ///   Sends a set of events to the associated Event Hub using a batched approach.  If the size of events exceed the
-        ///   maximum size of a single batch, an exception will be triggered and the send will fail.
+        ///   Sends a set of events to the associated Event Hub using a batched approach.  Unlike the <see cref="SendAsync(EventDataBatch, CancellationToken)" /> overload,
```

Comment:
> Unlike the see cref="SendAsync(EventDataBatch, CancellationToken)"  overload

Is this because the `SendEventOptions`, by definition, limit sending to a single partition? If so, perhapps that context would be valuable here and/or in the summary comments inside `SendEventOptions`

Reply:
Not at all; the difference is that the explicit batch has `TryAdd` semantics and prevents you from adding things that could fail where these overloads perform implicit batching and may fail when called.    I'll see if I can tune the verbiage a bit.

Reply:
Good call on that.  In hindsight, it definitely read awkwardly.   I removed the overload reference and, instead, concentrated on the point that validation can't be done upfront.